test/test_binary_ufuncs.py

```diff
@@ -2622,6 +2407,36 @@ def test_muldiv_scalar(self, device, dtype):
         self.assertEqual(x / s, x / y)
         self.assertEqual(s / x, y / x)
 
+    # TODO: update make_tensor to support extremal additions and remove this in favor of make_tensor
+    def _generate_input(self, shape, dtype, device, with_extremal):
+        if shape == ():
+            x = torch.tensor((), dtype=dtype, device=device)
```

Comment:
it's extremely conterintuitive to produce 0-element 1d tensor for shape=(), was scalar meant here?

Reply:
This is just copy-pasting the same function from earlier in the file and moving it closer to where it's used because it's legacy nonsense